jest.config.js

```diff
@@ -157,6 +157,9 @@ module.exports = {
   // This option allows use of a custom test runner
   // testRunner: "jasmine2",
 
+  // Default timeout of a test in milliseconds.
+  testTimeout: 10000,
```

Comment:
Changed the test suite config to extend the default timeout of 5s to 10s. This prevents test failures from slow local api calls and allows us to remove the overrides of the test timeout in individual test files.